Tests/MeiliSearchIntegrationTests/DocumentsTests.swift

```diff
@@ -81,26 +81,28 @@ class DocumentsTests: XCTestCase {
 
                 XCTAssertEqual(Update(updateId: 0), update)
 
-                Thread.sleep(forTimeInterval: 1.0)
+                waitPendingUpdate(self.client, self.uid, update) {
 
-                self.client.getDocuments(
-                    UID: self.uid,
-                    limit: 20
-                ) { (result: Result<[Movie], Swift.Error>) in
+                    self.client.getDocuments(
+                        UID: self.uid,
+                        limit: 20
+                    ) { (result: Result<[Movie], Swift.Error>) in
 
-                    switch result {
-                    case .success(let returnedMovies):
+                        switch result {
+                        case .success(let returnedMovies):
 
-                        movies.forEach { (movie: Movie) in
-                            XCTAssertTrue(returnedMovies.contains(movie))
+                            movies.forEach { (movie: Movie) in
+                                XCTAssertTrue(returnedMovies.contains(movie))
+                            }
+
+                        case .failure(let error):
+                            print(error)
+                            XCTFail()
```

Comment:
Missing XCTFail() message also i'm not sure the error is the one you recieve from MeiliSearch.

Reply:
`XCTFail()` is there to fail the test. `XCTFail(error)` does not print all the error, instead we use the `print(error)`.